data/strings/types_strings.txt

```diff
@@ -17109,6 +17110,8 @@
 
   [type.railway.station.subway.london]
     ref = type.railway.station.subway
+    en = London Underground Station
```

Comment:
As these stations are already drawn in London, there is little value to use the word London here.

Reply:
fixed